src/js/stores/BrandStore.jsx

```diff
@@ -54,8 +62,71 @@ class BrandStore extends EventEmitter {
       },
     });
   }
+ 
+  updateCard(card){
+    $.ajax({
+      url: "https://dev.calligre.com/api/info/card/" + card.id,
+      data : JSON.stringify(card),
+      type : 'PATCH',
+      contentType : 'application/json',
+      processData: false,
+      dataType: 'json',
+      headers: {
+        "Authorization": "Bearer " + AuthService.getToken()
+      },
+      success: function(response){
+        console.log(response);
```

Comment:
When we update the card are we also updating the entity in the store since we are not using the api response? Also, do we have any backend validation that might cause the data to go out of sync?

This question is for all the api calls in here, just got lazy and didn't want to retype it.